src/main/java/teammates/ui/controller/InstructorFeedbackAddAction.java

```diff
@@ -147,88 +143,19 @@ private void createTemplateFeedbackQuestions(String courseId, String feedbackSes
         return new ArrayList<FeedbackQuestionAttributes>();
     }
 
-    private FeedbackSessionAttributes extractFeedbackSessionData() {
-        //TODO assert parameters are not null then update test
-        //TODO make this method stateless
-        
-        FeedbackSessionAttributes newSession = new FeedbackSessionAttributes();
-        newSession.setCourseId(getRequestParamValue(Const.ParamsNames.COURSE_ID));
-        newSession.setFeedbackSessionName(Sanitizer.sanitizeTitle(
-                getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME)));
-        
+    @Override
+    protected FeedbackSessionAttributes extractFeedbackSessionDataHelper(FeedbackSessionAttributes newSession) {
         newSession.setCreatedTime(new Date());
-        newSession.setStartTime(TimeHelper.combineDateTime(
-                getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_STARTDATE),
-                getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_STARTTIME)));
-        newSession.setEndTime(TimeHelper.combineDateTime(
-                getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_ENDDATE),
-                getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_ENDTIME)));
-        String paramTimeZone = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_TIMEZONE);
-        if (paramTimeZone != null) {
-            newSession.setTimeZone(Double.parseDouble(paramTimeZone));
-        }
-        String paramGracePeriod = getRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_GRACEPERIOD);
-        if (paramGracePeriod != null) {
-            newSession.setGracePeriod(Integer.parseInt(paramGracePeriod));
-        }
-        
         newSession.setSentOpenEmail(false);
         newSession.setSentPublishedEmail(false);
-        
         newSession.setFeedbackSessionType(FeedbackSessionType.STANDARD);
```

Comment:
This is common to both this action and the other action. Can move to the abstract action instead?